addon-test-support/helpers.js

```diff
@@ -7,6 +7,13 @@ const DEFAULT_EVENT_OPTIONS = { canBubble: true, cancelable: true };
 const KEYBOARD_EVENT_TYPES = ['keydown', 'keypress', 'keyup'];
 const MOUSE_EVENT_TYPES = ['click', 'mousedown', 'mouseup', 'dblclick', 'mouseenter', 'mouseleave', 'mousemove', 'mouseout', 'mouseover'];
 
+// Reliable way to get the config?
```

Comment:
@rwjblue seems odd to require that the consumer set the root element but maybe that can work as a blueprint to add to the consuming app tests/helpers directory. I can try that. I could live with using the `window.requirejs.entries` as well :) but sure I'll try it.